<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>FIELD</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>691</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Lay_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Lay_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Lay_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Lay_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Lay_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Lay_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lay</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Num_4">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Num_5">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Num_6">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Num_8">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Num_9">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Num_10">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Place_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Place_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Place_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Place_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Place_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Place_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>COME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PASS LINE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DON'T PASS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
